misc/component.jsx

```diff
@@ -7,7 +7,7 @@ const MyComponent = ({ name, age, gender, happy, address, books, onBoom, color }
       {JSON.stringify(address)}
     </pre>
     <pre>
-      {JSON.stringify(books)}
+      {JSON.stringify(books.map(b => `--${b}`))}
```

Comment:
I wanna make sure it was an array that was printed and not a string or an object. So I used one of array prototype function to make sure of it